app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -56,7 +56,7 @@ import kotlin.coroutines.CoroutineContext
 class BrowserActivity : DuckDuckGoActivity(), CoroutineScope {
 
     override val coroutineContext: CoroutineContext
-        get() = SupervisorJob() + Dispatchers.Main
+        get() = job + Dispatchers.Main
```

Comment:
Instead of defining the `coroutineContext` manually, alongside a reference to a `job`, what's your thoughts on using `MainScope` instead?

## Class definition
`class BrowserActivity : DuckDuckGoActivity(), CoroutineScope by MainScope() {`

## Cancelling job
`coroutineContext.cancel()`

## Functionally equivalent?
I think this should be functionally equivalent, but would allow us to delete the `CoroutineContext` and `Job` references?